src/main/java/com/avispl/dal/communicator/cisco/dto/ValueSpaceRefHolder.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2021 AVI-SPL Inc. All Rights Reserved.
+ */
+package com.avispl.dal.communicator.cisco.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.xml.bind.annotation.*;
+import java.util.ArrayList;
+import java.util.List;
+
+/***
+ * Representation of nodes, that contain values, item and valueSpaceRef attrubutes, example
+ * <DviType valueSpaceRef="/Valuespace/TTPAR_DviType">AutoDetect</DviType>
+ * or
+ * <Connector item="2" maxOccurrence="2"></Connector>
+ *
+ * valueSpaceRef is later used for fetching additional value details from /valuespace.xml, to define control limits,
+ * controllable property type etc.
+ *
+ * @author Maksym.Rossiitsev / Symphony Dev Team<br>
+ * Created on Apr 26, 2021
+ * @since 1.0
+ */

```

Comment:
special characters such as '<' will result in Symphony's javadoc generator to fail. You should use '&lt;'.
Or, you can just wrap any special characters in {@code ..... }, for example:
{@code <DviType valueSpaceRef="/Valuespace/TTPAR_DviType">AutoDetect</DviType>}